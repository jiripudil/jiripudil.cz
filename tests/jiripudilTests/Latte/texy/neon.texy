## Title of the document

Some paragraph.

/--neon
extensions:
	- FooExtension(%wwwDir%)
\--
