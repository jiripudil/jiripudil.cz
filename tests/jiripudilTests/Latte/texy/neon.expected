<h2>Title of the document</h2>

<p>Some paragraph.</p>

<pre><code><span class="neon-section">extensions</span><span
class="neon-sep">:</span><span class="neon-section">
    </span><span
class="neon-sep">-</span><span class="neon-value"> FooExtension(</span><span
class="neon-var">%wwwDir%</span><span class="neon-value">)</span></code></pre>
